app/code/Magento/InventoryImportExport/Helper/SourceItemConvert.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryImportExport\Helper;
+
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+use Magento\InventoryApi\Api\Data\SourceItemInterfaceFactory;
+use Magento\InventoryImportExport\Model\Import\Sources;
+
+class SourceItemConvert
```

Comment:
no need to Introduce Helper in the new Code.

Helpers should be discouraged in newly created code.
You could create SourceItemConvert under the Model namespace instead